markdown/events/2021/bytesize-15-graphic-design.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Bytesize 15: Graphic design / pipeline diagrams"
+subtitle: Zandra Fagernäs - LMU Münich / MPI-EVA
```

Comment:
```suggestion
subtitle: Zandra Fagernäs - LMU Munich / MPI-EVA
```

Either Munich (english) or München (german) ;-)

Reply:
I copied this from @jfy133 page! 😅

Reply:
:sweat_smile: Germlish...